pages/ksphere/konvoy/1.5/install/install-aws/advanced-provisioning/index.md

```diff
@@ -512,7 +512,7 @@ To add custom resource files:
 [aws_instance_types]: https://aws.amazon.com/ec2/instance-types/
 [ephemeral_storage]: ../../../storage/
 [ebs_volume_types]: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html
-[cloud_provider]: https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/
+[cloud_provider]: https://v1-17.docs.kubernetes.io/docs/concepts/cluster-administration/cloud-providers/
```

Comment:
This link points to documentation that is no longer maintained and is static. I don't think it's a good idea to point to old docs. Can we find a different way to solve this broken link?

Reply:
The page `cloud-providers` no longer exists in the latest k8s docs so we likely have to delete this link if we don't want to use the static one.

Reply:
I don't get why though. I guess, for now, let's leave this link in there. And then I'll make a note in JIRA to reevaluate this at some point. Probably a low priority with all the other things.